Utilities/StorageFactory/test/test_file_statistics_sender.sh

```diff
@@ -3,6 +3,22 @@
 # Pass in name and status
 function die { echo $1: status $2 ;  exit $2; }
 
+function testJSON {
+    grep cmssw_version $1 > test.json
```

Comment:
Instead of testing only cmssw_version, what about testing the other entries as well?

Reply:
This line extracts the JSON documents from the log file into a separate file. I could have used any other key as well, or e.g. `{", but went with `cmssw_version` for no particual reason. Do you have a suggestion how to do it better? Or would it be good to add a comment here explaining this?

Reply:
Added comment